packages/aila/src/protocol/jsonPatchProtocol.ts

```diff
@@ -612,23 +612,25 @@ export function tryParsePart(
 ): MessagePartDocument | UnknownDocument {
   const { type } = obj;
 
-  // Assert the message part type is allowed
   if (!MessagePartDocumentSchemaByType[type as MessagePartType]) {
-    log.error("Invalid message part type", type);
     return {
-      type: "unknown",
-      value: JSON.stringify,
+      type: "unknown" as const,
+      value: JSON.stringify(obj),
```

Comment:
This seems to be a bug? It was sending the JSON.stringify method as the value?

Reply:
That sure looks like a bug! I suppose it doesn't really have a downstream consequence or we would have noticed it sooner